config/absearch.ini

```diff
@@ -65,9 +65,11 @@ num_retries = 1
 
 
 [statsd]
+enabled = 1
 host = localhost
 port = 8125
 prefix = absearch
+count_cohorts_enabled = 0
```

Comment:
We might change elsewhere as well if that is what bothers you :+1: (we shall probably put them in lowercase though)